tests/components/ozw/test_light.py

```diff
@@ -149,3 +149,335 @@ async def test_light(hass, light_data, light_msg, sent_messages):
+    assert msg["payload"] == {"Value": "#ff99000000", "ValueIDKey": 659341335}
+
+    # Feedback on state
+    light_msg.decode()
+    light_msg.payload["Value"] = byte_to_zwave_brightness(255)
+    light_msg.encode()
+    light_rgb_msg.decode()
+    light_rgb_msg.payload["Value"] = "#ff99000000"
+    light_rgb_msg.encode()
+    receive_message(light_msg)
+    receive_message(light_rgb_msg)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["rgb_color"] == (255, 153, 0)
+
+    # Test setting xy_color
+    new_color = [0.52, 0.43]
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.led_bulb_6_multi_colour_level", "xy_color": new_color},
+        blocking=True,
+    )
+    assert len(sent_messages) == 15
+    msg = sent_messages[-1]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 659128337}
+
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#ffbb370000", "ValueIDKey": 659341335}
+
+    # Feedback on state
+    light_msg.decode()
+    light_msg.payload["Value"] = byte_to_zwave_brightness(255)
+    light_msg.encode()
+    light_rgb_msg.decode()
+    light_rgb_msg.payload["Value"] = "#ffbb370000"
+    light_rgb_msg.encode()
+    receive_message(light_msg)
+    receive_message(light_rgb_msg)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["xy_color"] == (0.519, 0.429)
+
+    # Test setting color temp
+    new_color = 465
+    await hass.services.async_call(
+        "light",
+        "turn_on",
+        {"entity_id": "light.led_bulb_6_multi_colour_level", "color_temp": new_color},
+        blocking=True,
+    )
+    assert len(sent_messages) == 17
+    msg = sent_messages[-1]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": 255, "ValueIDKey": 659128337}
+
+    msg = sent_messages[-2]
+    assert msg["topic"] == "OpenZWave/1/command/setvalue/"
+    assert msg["payload"] == {"Value": "#000000e51a", "ValueIDKey": 659341335}
+
+    # Feedback on state
+    light_msg.decode()
+    light_msg.payload["Value"] = byte_to_zwave_brightness(255)
+    light_msg.encode()
+    light_rgb_msg.decode()
+    light_rgb_msg.payload["Value"] = "#000000e51a"
+    light_rgb_msg.encode()
+    receive_message(light_msg)
+    receive_message(light_rgb_msg)
+    await hass.async_block_till_done()
+
+    state = hass.states.get("light.led_bulb_6_multi_colour_level")
+    assert state is not None
+    assert state.state == "on"
+    assert state.attributes["color_temp"] == 465
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5844602)